R/get_ssm_by_region.R

```diff
@@ -19,18 +19,19 @@
 #' @param streamlined Return Start_Position and Tumor_Smaple_Barcode as the only two MAF columns. Default is FALSE.
 #' @param projection Obtain variants projected to this reference (one of grch37 or hg38).
 #' @param this_seq_type The seq_type you want back, default is genome.
+#' @param mode Accepts 2 options of "slms-3" and "publication" to indicate which variant calls to use. Default is "slms-3".
```

Comment:
Would this be something like `Accepts one of the 2 options "slms-3" and "publication ...`?

Should this description be harmonized with `tool_name`'s description?

Reply:
Will add this, thanks!

Reply:
this has bee added :heavy_check_mark: